packages/application-components/src/experimental/components/internals/centered-loading-spinner/centered-loading-spinner.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import LoadingSpinner from '@commercetools-uikit/loading-spinner';
+
+const CenteredLoadingSpinner = (): JSX.Element => (
+  <div
+    data-testid="picker-centered-loading-spinner"
```

Comment:
In the future (not sure worth doing now) the file can probably be moved outside of the `internals` scope and the `test-id` changed to be just `data-testid="centered-loading-spinner"`. I can foresee the need for this component in other places too.

Reply:
yes I have always considered this, but I came to the same conclusion as you that this is premature